.github/workflows/test_gen.yaml

```diff
@@ -454,10 +469,18 @@ jobs:
       uses: actions/download-artifact@v4
       with:
         name: python-${{ matrix.python-version }}-pytest.xml.3
-    - name: Download integration test results (databricks-unity)
+    - name: Download integration test results (databricks-unity_1)
+      uses: actions/download-artifact@v4
+      with:
+        name: python-${{ matrix.python-version }}-pytest.xml.4.1
+    - name: Download integration test results (databricks-unity_2)
+      uses: actions/download-artifact@v4
+      with:
+        name: python-${{ matrix.python-version }}-pytest.xml.4.2
+    - name: Download integration test results (databricks-unity_3)
       uses: actions/download-artifact@v4
       with:
-        name: python-${{ matrix.python-version }}-pytest.xml.4
+        name: python-${{ matrix.python-version }}-pytest.xml.4.3
```

Comment:
I wonder if it's possible to glob multiple files with a pattern so we don't need to repeat this for every split (and actually even for every platforms)?

Reply:
Testing it out now